src/dune_rules/pkg_config.ml

```diff
@@ -1,5 +1,11 @@
 open Import
 
+let pkg_config_binary sctx =
+  let env = Super_context.context_env sctx in
+  match Env.get env "PKG_CONFIG" with
```

Comment:
We should probably allow overriding this variable. I recommend fetching the environment from `Super_context.env_node`

Reply:
Ah yeah, env_node makes sense. I changed to that.